File: examples/src/Presentations/Conversion/ExportShapeToSVG.cs
import aspose.slides as slides
using Aspose.slides.Examples.CSharp
using Aspose.slides.Export
using System
using System.Collections.Generic
using System.IO
using System.Linq
using System.text
using System.Threading.Tasks

namespace CSharp.Presentations.Conversion
{
	class ExportShapeToSVG
	{
		#ExStart:ExportShapeToSVG
		public static void Run()
		{
			
			outSvgFileName = "SingleShape.svg"
			dataDir = RunExamples.GetDataDir_Conversion()
			using (Presentation pres = new Presentation(dataDir+ "TestExportShapeToSvg.pptx"))
			{
				using (Stream stream = new FileStream(outSvgFileName, FileMode.Create, FileAccess.Write))
				{
					pres.slides[0].shapes[0].WriteAsSvg(stream)

					
				}
			
				
			}


		}


		#ExEnd:ExportShapeToSVG
	}
}

File: examples/src/Presentations/Conversion/export_shape_to_svg.py
import aspose.slides as slides


def export_shape_to_svg(global_opts):
	with slides.Presentation(global_opts.data_dir + "welcome-to-powerpoint.pptx") as pres:
		with open(global_opts.out_dir + "export_shape_to_svg_out.svg", "wb") as stream:
			pres.slides[0].shapes[0].write_as_svg(stream)
